chart/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -142,7 +142,7 @@ data:
     {{- if .Values.featureFlags.kubeappsAPIsServer }}
       location ~* /apis {
         rewrite ^ $request_uri; # pass the encoded url downstream as is,
-        rewrite /apis([^?]*) $1?$args break;
+        rewrite /apis([^?]*) $1 break;
```

Comment:
Why this change?

Also, I was originally wondering why this is relevant at all, since it's the url for handling the gateway rendered API endpoints. I was expecting to see something for forwarding requests via content-type for the actual grpc/grpc-web requests, but I see down further that the client in the dashboard is configured to talk to `/apis` as a base URL. Nice, that's a bit neater in that we don't need nginx to do the proxying specifically.